src/Entities/Player.cpp

```diff
@@ -934,17 +934,18 @@ void cPlayer::SetFlying(bool a_IsFlying)
 
 
 
-void cPlayer::ApplyArmorDamage(int DamageBlocked)
+void cPlayer::ApplyArmorDamage(int a_DamageBlocked)
 {
-	short ArmorDamage = static_cast<short>(DamageBlocked / 4);
+	short ArmorDamage = static_cast<short>(a_DamageBlocked / 4);
```

Comment:
> Any hit from a damage source that can be blocked by armor will remove one point of durability from each piece of armor worn for every 4 () of incoming damage (rounded down, **but never below 1**).

(https://minecraft.gamepedia.com/Armor#Damage_types)

Reply:
Yes, that is what these lines do. Am I missing something?

Reply:
You're missing `std::min`. Otherwise damages of `3` or lower would become `0`.

Reply:
The next lines change the value to 1 if it's 0, but `std::min` is indeed better, I will change it.

Reply:
Oh, just overlooked the following if.